ethereum_history_api/circuits/lib/src/fixtures/mainnet/cancun/small_block/transaction.nr

```diff
@@ -0,0 +1,22 @@
+use crate::transaction::Transaction;
+
+global transaction = Transaction {
+  nonce: 0x485a,
```

Comment:
Let's optimise for readability and have numeric fields as numbers. On etherscan nonce, gas price & gas limit are numeric.